src/task/script.js

```diff
@@ -389,20 +393,32 @@ function clearUserNotFound() {
 function createSuggestionsList(matches) {
   const listItems = document.getElementById('list-items');
   if (matches.length) {
-    matches.map(({ username }) => {
+    assigneeAvatarLoader.style.display = 'initial';
+    matches.map(({ username, picture = {} }) => {
+      const defaultImagePath = '../images/avatar.png';
+      const imageUrl =
+        picture.hasOwnProperty('url') && picture.url
+          ? picture.url
+          : defaultImagePath;
       const listItem = document.createElement('p');
       listItem.classList.add('list-item');
       listItem.style.cursor = 'pointer';
-      listItem.setAttribute('onclick', `setAssignee('${username}')`);
+      listItem.setAttribute(
+        'onclick',
+        `setAssignee('${username}', '${imageUrl}')`,
+      );
       listItem.innerText = username;
       listItems.appendChild(listItem);
     });
   }
 }
 
-function setAssignee(assignee) {
+function setAssignee(assignee, imgUrl) {
   assigneeEl.value = assignee;
   wasAssigneeSet = true;
+  assigneeAvatar.style.display = 'initial';
+  assigneeAvatarLoader.style.display = 'none';
```

Comment:
I see there is the reusable task:

1. Hide/show of the elements or content
2. Update of the image
3. Creating element

Move all these to a reusable method  rather than re-creating the same code again and again